app/controllers/application_controller/show.rb

```diff
@@ -0,0 +1,29 @@
+module ApplicationController::Show
+  extend ActiveSupport::Concern
+  def show_timeline(breadcrumb_url = nil)
```

Comment:
I :+1:  the direction of this. 
Please note though that now there are 3 identical (or very very similar methods) for this - here, in containers and in vm_common.rb